lib/internal/test_runner/reporter/dot.js

```diff
@@ -1,17 +1,26 @@
 'use strict';
+const { MathMax } = primordials;
 
 module.exports = async function* dot(source) {
   let count = 0;
+  let columns = getLineLength();
   for await (const { type } of source) {
     if (type === 'test:pass') {
       yield '.';
     }
     if (type === 'test:fail') {
       yield 'X';
     }
-    if ((type === 'test:fail' || type === 'test:pass') && ++count % 20 === 0) {
+    if ((type === 'test:fail' || type === 'test:pass') && ++count % columns === 0) {
```

Comment:
we cannot use modulo here if we want to support terminal resize.

Reply:
thanks, I saw that and in the last commit I fixed it, can you re-review?